graph/formats/rdf/urna_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package rdf
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestURNA(t *testing.T) {
+	glob, err := filepath.Glob(filepath.Join("testdata", *tests))
+	if err != nil {
+		t.Fatalf("Failed to open test suite: %v", err)
+	}
+	for _, test := range []struct {
+		name  string
+		fn    func(dst, src []*Statement) ([]*Statement, error)
+		truth string
+	}{
+		{
+			name:  "URDNA2015",
+			fn:    URDNA2015,
+			truth: "-urdna2015.nq",
+		},
+		{
+			name:  "URGNA2012",
+			fn:    URGNA2012,
+			truth: "-urgna2012.nq",
+		},
+	} {
+		t.Run(test.name, func(t *testing.T) {
+			for _, path := range glob {
+				name := filepath.Base(path)
+				golden := strings.TrimSuffix(path, "-in.nq") + test.truth
+				want, err := ioutil.ReadFile(golden)
```

Comment:
`TestURNA` fails on Windows because of `\r\n` line endings that appear in "want" and not in "got". A fix following this line could be `want = bytes.ReplaceAll(want, []byte("\r"), nil)` or something to that effect.

Reply:
How are you cloning the repo? We assume autocrlf == false because any other behaviour is evil.

Reply:
I use `git clone git@github.com:gonum/gonum.git`. Yeah that was it, `autocrlf` was blank. I'll approve now.

Reply:
I think the default behaviour on windows is autcrlf=true.